.github/workflows/pull-request.yml

```diff
@@ -159,10 +159,13 @@ jobs:
             coverage/
             junit.xml
 
-      - name: Run Storybook Tests
-        # We want to enforce that the actual `turbo@latest` package is used instead of a possible hijack from the user
-        # the `${{ needs.base.outputs.turbo_args }}` is a string substitution happening from the base job
-        run: npx --package=turbo@latest -- turbo test:storybook:local ${{ needs.base.outputs.turbo_args }} -- --ci
+      - name: Run Storybook Visual Regression Tests
+        # We use Chromatic for Visual Regression Testing
+        # Note that this token is not a secret and safe to be added directly on the CI
+        # This Step wll fail if there are Visual Regressions and then should be acted on on Chromatic's UI or Review Comments
+        # See https://www.chromatic.com/docs/review
+        # If you're a Node.js Website Collaborator and do not have access to Chromatic, please request access to the Website Team Maintainers
+        run: npx --package=chromatic@latest -- chromatic --project-token=chpt_17c2d0dc8a68711 --build-script-name=storybook:build --only-changed --ci
```

Comment:
Why the not the [chromatic action](https://github.com/marketplace/actions/publish-to-chromatic)?

Reply:
No real need to be honest. In the end it does the same thing, or do you see any benefit over invoking the package? (I’m fine either way)